lib/msf/ui/console/command_dispatcher/post.rb

```diff
@@ -166,6 +166,13 @@ def cmd_run_tabs(str, words)
 
   alias cmd_exploit_tabs cmd_run_tabs
 
+  #
+  # Tab completion for the run command
+  #
+  def cmd_run_tabs(str, words)
```

Comment:
This function is already defined above, it looks like the functionality should be merged :+1:

Reply:
Ah, my bad. Mustn't have seen that. I will merge those.